plugins/ipam/ipam.go

```diff
@@ -1061,3 +1068,98 @@ func customIfID(ifName, network string) string {
 	}
 	return ifName + "/" + network
 }
+
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
You can get rid of this comment, it does no need to be marked this way.
Feel free to modify the copyright header of the source files you touched.

Reply:
I have modified the copyright header. Feel free to review if it is correct?